.storybook/docs/WebComponentAttrs.tsx

```diff
@@ -0,0 +1,49 @@
+import humanizeList from 'humanize-react';
+import { useOf } from '@storybook/blocks';
+
+interface WebComponentAttrsProps {
+  of?: any;
+}
+
+/**
+ * If the component we're documenting passes extra props through to an HTML element,
+ * we want to include documentation about that. Rather than listing out all the
+ * additional attributes for those HTML elements in our props table, we will add
+ * a link to the Github file naming these attributes. Looks for a `underlyingAttrLists`
+ * array in `story.parameters.docs` to determine if it should show this additional
+ * text and what elements' documentation it should link to.
+ */
+export const WebComponentAttrs = ({ of }) => {
+  const resolvedOf = useOf(of || 'story', ['story', 'meta']);
+  if (resolvedOf.type !== 'story') {
+    return null;
+  }
+
+  const elements = resolvedOf.story.parameters?.docs?.underlyingAttrLists;
```

Comment:
The "underlying" part has perhaps lost its meaning. Naming suggestion: `sharedAttributesLists`